src/sisp3DS/sisp.js

```diff
@@ -0,0 +1,250 @@
+   * @param {String} userBillingInfo.billAddrCountry - Required - The country of the billing address of the card holder, it should be the first 3 digits of the ISO 3166-1
+   * @param {String} userBillingInfo.billAddrLine1 - Optional - First billing address of the card holder
+   * @param {String} userBillingInfo.billAddrLine2 - Optional - Second billing address of the card holder
+   * @param {String} userBillingInfo.billAddrLine3 - Optional - Thrid billing address of the card holder
+   * @param {String} userBillingInfo.billAddrPostCode - Optional - Postal code of the billing address of the card holder
+   * @param {String} userBillingInfo.billAddrState - Optional - State of the billing address of the card holder. It should be the sub-division of the defined country in ISO 3166-2
+   * @param {String} userBillingInfo.shipAddrCity - Optional - City of the delivery address of the card holder
+   * @param {String} userBillingInfo.shipAddrCountry - Optional - The country of the delivery address of the card holder, it should be the first 3 digits of the ISO 3166-1
+   * @param {String} userBillingInfo.shipAddrLine1 - Optional - First delivery address of the card holder
+   * @param {String} userBillingInfo.shipAddrPostCode - Optional - Postal code of the delivery address of the card holder
+   * @param {String} userBillingInfo.shipAddrState - Optional - State of the delivery address of the card holder. It should be the sub-division of the defined country in ISO 3166-2
+   * @param {Object} userBillingInfo.workPhone - Optional - An Object containing information about work phone number of the card holder
+   * @param {String} userBillingInfo.workPhone.cc - Required - This indicates the country code of the country of the phone number, example - 123
+   * @param {String} userBillingInfo.workPhone.subscriber - Required - This indicates the phone number of the account holder, it should include the indicator without the + sign, example of a phone number of Cape Verd: 2389573234
+   * @param {Object} userBillingInfo.mobilePhone - Required - An Object containing information about personal phone number of the card holder
+   * @param {String} userBillingInfo.mobilePhone.cc - Required - This indicates the country code of the country of the phone number, example example - 123
+   * @param {String} userBillingInfo.mobilePhone.subscriber - Required - This indicates the phone number of the account holder, it should include the indicator without the + sign, example of a phone number of Cape Verd: 2389573234
+   * @param {String} options.languageMessages - Optional - This is used to setup the languages for the response messages.
+   * @param {Object} options.currencyCode - Optional - This is to setup the currency.
+   * @returns {Document} response - HTML Form to process payments
+   */
+  generatePaymentRequestForm = (referenceId, total, webhookUrl, userBillingInfo, options = { currencyCode: '', languageMessages: '' }) => {
```

Comment:
See which parameters above can be better documented as enum types. I don't know what a valid currency code is, for example. `userBillingInfo.addrMatch` is another example.